lib/platform/bitbucket/__snapshots__/index.spec.ts.snap

```diff
@@ -1051,7 +1051,7 @@ Array [
       "user-agent": "https://github.com/renovatebot/renovate",
     },
     "method": "GET",
-    "url": "https://api.bitbucket.org/2.0/repositories/?role=contributor&pagelen=100",
+    "url": "https://api.bitbucket.org/2.0/repositories?role=contributor&pagelen=100",
```

Comment:
Shouldn't be breaking, test run performed well for me

Reply:
change should be fine. i think previous url was wrong anyways